src/chrome/content/rules/Interpol.int.xml

```diff
@@ -0,0 +1,33 @@
+<!-- 
+	Non-functional:
```

Comment:
Please double check them, at least `cyber.`, `s.` and `secure.` are working for me.
Could you also add the type of error those hosts have with HTTPS?